app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -195,8 +225,15 @@ private void updateImage(Media media) {
 
     public void swipeToNext() {
         int nextPos = reviewPager.getCurrentItem() + 1;
+        // If currently at category fragment, then check if category exists
```

Comment:
`check if category exists` could be ambiguous for a newcomer.
I suggest `check whether the media has any non-hidden category`